go/cmd/vtcombo/tablet_map.go

```diff
@@ -384,3 +371,257 @@ func (itc *internalTabletConn) StreamHealth(ctx context.Context) (<-chan *queryp
 		return tabletconn.TabletErrorFromGRPC(tabletserver.ToGRPCError(finalErr))
 	}, nil
 }
+
+//
+// TabletManagerClient implementation
+//
+
+// internalTabletManagerClient implements tmclient.TabletManagerClient
+type internalTabletManagerClient struct{}
+
+func (itmc *internalTabletManagerClient) Ping(ctx context.Context, tablet *topo.TabletInfo) error {
```

Comment:
already have a different class in the previous section, I wanted the receiver to be different too, and easy to search for.